src/VirtoCommerce.SimpleExportImportModule.Web/Scripts/blades/file-upload.js

```diff
@@ -0,0 +1,120 @@
+angular.module('virtoCommerce.simpleExportImportModule')
+.controller('virtoCommerce.simpleExportImportModule.fileUploadController',
+    ['FileUploader', '$document', '$scope', '$timeout', 'platformWebApp.bladeNavigationService', 'platformWebApp.assets.api',
+    function(FileUploader, $document, $scope, $timeout, bladeNavigationService, assetsApi) {
+        const blade = $scope.blade;
+        const maxCsvSize = 1048576; // 1024 * 1024 | Math.pow(2,20); | 0x100000
```

Comment:
Why not calculate?

Reply:
Why do we need to calculate it? (since it's a constant) The comment was added just for clarification, because if you google 1mb to bytes, you'll get this 
![Снимок экрана 2021-03-25 в 12 41 05](https://user-images.githubusercontent.com/22875828/112460281-63c5fe00-8d67-11eb-8842-596b3913614b.png)

Reply:
Just because
```js
const oneKb = 1024;
const oneMb = 1024 * kb;
const maxCsvSize = oneMb;
```
is much cleaner to understand. But at your discretion

Reply:
done

Reply:
(and yep, google is right, 1024*1024 is mebibyte, not mega, if follow SI)